### PR TITLE
Update suricata.rb

### PR DESCRIPTION
**Pull request by missdiog**
- [yes ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [ yes] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ yes] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ yes] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

update suricata to 3.1.2.
support hiredis,nss,nspr

### CODE REVIEW

File: Formula/suricata.rb [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
   desc "Network IDS, IPS, and security monitoring engine"
   homepage "https://suricata-ids.org/"
-  url "https://www.openinfosecfoundation.org/download/suricata-2.0.8.tar.gz"
-  sha256 "7af6394cb81e464f5c1ac88a1444030e30940caab6e53688a6d9eb652226d1be"
+  url "https://www.openinfosecfoundation.org/download/suricata-3.1.2.tar.gz"
+  sha256 "f9e7742580849f202254e75d9fc245ba53f4d7490f47a6d30f02a7b10aacc512"
 
   bottle do
-    rebuild 1
+    revision 1
```
Comment (bfontaine) [inline Formula/suricata.rb]: Please don’t change this line; our bot will update it. Thanks!

**Review by bfontaine — CHANGES_REQUESTED** [review]

**Comment by dunn** [comment]
@BrewTestBot test this please

**Review by bfontaine — APPROVED** [review]

**Comment by dunn** [comment]
Still getting a checksum mismatch on the bottles; let's try again.

**Comment by MikeMcQuaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

For future reference the preferred commit message format is `suricata 3.1.2, 3.2beta1 (devel)`. Please complete the issue template in future PRs where this format is detailed in the linked [Contributing guidelines](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md). Thanks!

**Pull request closed without merging**